src/config.js

```diff
@@ -60,8 +63,8 @@ module.exports = {
       state: true,
     },
   },
-  jwtSecret:
-    'eAwI4zcTDd4Pvc8QtN9z57Fqsr4ENNcTpK1x4A1dCLj0Y44OravXZDzNbA-4VEwAIh1Hw3vn1nhB9ygWLqAGE4GiX6hjjLsJi8IJ',
+  // eslint-disable-next-line prettier/prettier
+  jwtSecret: 'eAwI4zcTDd4Pvc8QtN9z57Fqsr4ENNcTpK1x4A1dCLj0Y44OravXZDzNbA-4VEwAIh1Hw3vn1nhB9ygWLqAGE4GiX6hjjLsJi8IJ',
```

Comment:
😱

Reply:
didn't bother figuring out how to find+replace 2 lines with sed lol.